### PR TITLE
Updating to 6.1 SF release.

### PR DESCRIPTION
**Pull request by amanbha**
Summary of changes:
- Merging code for SF 6.1 release
- Adding dotnet core projects.
- Converting existing .csproj to new project format.
- Refactoring build to enable dotnetcore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Please refer to [Readme.md](https://github.com/Azure/service-fabric/blob/master/
 ## Releases and Support
 Official releases from Microsoft of the NuGet packages in this repo are released directly to NuGet and Web Platform Installer. Get the latest official release [here](http://www.microsoft.com/web/handlers/webpi.ashx?command=getinstallerredirect&appid=MicrosoftAzure-ServiceFabric-VS2015).
```
Comment (hrushib) [inline README.md]: Release link above doesn't seem to work. Also we should replace above link by the one for VS 2017.

File: build.ps1 [hunk 2]
```diff
@@ -25,7 +25,7 @@ param
 
 
 
-# Check msbuild exists
+# Check msbuild exists. Find msbuild for VS2017
 $msbuildPath = "MSBuild\14.0\bin\MSBuild.exe"
```
Comment (hrushib) [inline build.ps1]: Above line should be removed.

File: src/Microsoft.ServiceFabric.Actors/ActorExtensions.cs [hunk 3]
```diff
@@ -10,7 +11,7 @@ namespace Microsoft.ServiceFabric.Actors
     public static class ActorExtensions
     {
         /// <summary>
-        /// Gets <see cref="ActorId"/> for the actor.
+        /// Gets <see cref="ActorId"/> for the actor./>
```
Comment (hrushib) [inline src/Microsoft.ServiceFabric.Actors/ActorExtensions.cs]: Remove mistakenly added characters /> from above line...

File: build.ps1 (hunk 2 above)
Comment (amanbha) [reply build.ps1]: Thanks, fixed

File: src/Microsoft.ServiceFabric.Actors/ActorExtensions.cs (hunk 3 above)
Comment (amanbha) [reply src/Microsoft.ServiceFabric.Actors/ActorExtensions.cs]: Thanks, fixed

File: README.md (hunk 1 above)
Comment (amanbha) [reply README.md]: Thanks, Link is working, we only release VS 2015 tooling in our SDK, VS 2017 SF tools are part of VS 2017 release.

**Pull request merged**